include/linux/soundwire/sdw.h

```diff
@@ -888,6 +889,7 @@ struct sdw_master_ops {
  * @multi_link: Store bus property that indicates if multi links
  * are supported. This flag is populated by drivers after reading
  * appropriate firmware (ACPI/DT).
+ * @lane_used_bandwidth: how much bandwidth is used by each lane
```

Comment:
probably need to state the unit, I guess this is bits per second?

Reply:
> probably need to state the unit, I guess this is bits per second?

Yes, it is m_rt->stream->params.rate * hweight32(m_p_rt->ch_mask) * m_rt->stream->params.bps.